### PR TITLE
Automatic reload on list update

### LINKED ISSUE #4: Automatically update redirectUrl
Instead of manually reload the app with the "Reload" button, 
redirectUrl should be automatically updated every time a user adds a new URL.

### PR DESCRIPTION
**Pull request by pchatard**
Resolves #4 :

- Reload the app when the websites list is updated (add or delete).
- On reload, keep track of the current value of the timer to avoid setting it back to zero if a website is added or removed.
- Update the UI by removing the reload button.

### CODE REVIEW

**Comment by Icesofty** [comment]
Thanks for your PR, 
I'll review it later. 

> Resolves #4 :
> 
> * Reload the app when the websites list is updated (add or delete).
> * On reload, keep track of the current value of the timer to avoid setting it back to zero if a website is added or removed.
> * **Update the UI by removing the reload button.**

Can you re-implement the Reload button? By changing the name to "Reset" or "Reset Timer" maybe?

**Comment by pchatard** [comment]
> Can you re-implement the Reload button? By changing the name to "Reset" or "Reset Timer" maybe?

Sure, it's done. The button now resets the timer to zero.

File: app/popup.js [hunk 1]
```diff
@@ -143,7 +155,8 @@ document.getElementById('itemlist').addEventListener('click', function (e) {
   if (tgt.tagName.toUpperCase() == 'LI') {
     tgt.parentNode.removeChild(tgt); // or tgt.remove();
   }
+  reload();
```
Comment (Icesofty) [inline app/popup.js:158]: This is smart.

Comment (Icesofty) [inline app/popup.js:158]: Can be a UX issue, but this is smart.

**Review by Icesofty — APPROVED** [review]

**Review by arthureichelberger — APPROVED** [review]

**Pull request merged**